src/api/slack/chat.ts

```diff
@@ -8,5 +8,8 @@ export interface IPostMessageParams {
 
 export const postMessage = async (params: IPostMessageParams) => {
     const web = new WebClient(SLACK_TOKEN);
-    return await web.chat.postMessage(params);
+    return await web.chat.postMessage({
```

Comment:
What's the reason for this change?

Reply:
No idea, vscode said that it was an error and couldn't do the conversion and the docs said to do it like that and it fixed the error :shrug: